src/main/java/net/mcreator/ui/MCreator.java

```diff
@@ -430,9 +430,9 @@ public MainToolBar getToolBar() {
 		return toolBar;
 	}
 
-	public void removeStoreFiles(File folder) {
+	public void removeStoreFiles() {
 		try {
-			try (var paths = Files.walk(folder.toPath())) {
+			try (var paths = Files.walk(getWorkspace().getWorkspaceFolder().toPath())) {

```

Comment:
Before you were using:
```java
				actionRegistry.getMCreator().removeStoreFiles(actionRegistry.getMCreator().getWorkspaceFolder());
```
Why not do the same here?
```suggestion
			try (var paths = Files.walk(getWorkspaceFolder().toPath())) {
```

Reply:
Uh, nevermind, did not notice you have already removed this part.